CrisisTextLine/Elmer
# Newrelic

#### What's this PR do?
adds new relic monitoring to cron jobs
#### Where should the reviewer start?
any script in job file and runJobs.js
#### How should this be manually tested?
run on your local, check newrelic, you should see your local machine as one of the hosts and see job run times in transactions
#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/browse/INT-148
#### Questions:
